web/viewWorkingHoursResultsReport.php

```diff
@@ -145,6 +146,10 @@ function populate() {
                     var index = store.findExact('login', selectedLogin);
                     grid.getSelectionModel().selectRow(index);
                     grid.getView().focusRow(index);
+                } else {
+                    loggedInUserIndex = store.findExact('login', loggedInUser);
+                    grid.getSelectionModel().selectRow(loggedInUserIndex);
+                    grid.getView().focusRow(loggedInUserIndex);
```

Comment:
If you do this, remember to remove the `selectedLogin = undefined;` assignment in the workingResultsForm view listener some lines below!